scripts/jira-automation.py

```diff
@@ -361,15 +453,18 @@ def get_jira_description(self, snyk_org_slug: str, snyk_project_id: str) -> str:
             f"Vulnerability URL: {self.get_url()}. \n\n"
             f"CSSV score: {self.get_cvss_score()}. \n\n"
             f"CVE Identifier: {cve}. \n\n"
+            f"CWE Identifier: {cwe}. \n\n"
             f"More info can be found in https://app.snyk.io/org/{snyk_org_slug}/project/{snyk_project_id}#issue-{self.get_id()}. \n"
```

Comment:
isn't the full url returned as part of the response?

Reply:
AFAIK, no I will take another look in anycase

Reply:
the difference is 
the url returned by snyk API is the general explanation of the vuln.
The custom one created here will point to the issue itself something like: https://app.snyk.io/org/red-hat-openshift-data-science-rhods/project/11c5398c-8c13-4662-8877-41ada4f7c956#issue-200cee3b-f84b-4e4a-8059-d9b2fde35b1d